lib/active_triples/relation.rb

```diff
@@ -11,13 +11,18 @@ module ActiveTriples
   # Relations, then, express n binary relationships between the parent node and
   # a term.
   #
+  # 
+  #
   # @see RDF::Term
   class Relation
-
+    include Enumerable
+    
     attr_accessor :parent, :value_arguments, :node_cache, :rel_args
     attr_reader :reflections
 
-    delegate *(Array.public_instance_methods - [:send, :__send__, :__id__, :class, :object_id] + [:as_json]), :to => :result
+    # delegate *(Array.public_instance_methods - [:send, :__send__, :__id__, :class, :object_id] + [:as_json]), :to => :result
+    delegate :[], :each, :empty?, :to_a, :to_ary, :<=>, :==, :equal, :===, :last, 
```

Comment:
Is this all we need? I really don't know - does Enumerable define things like #length?